src/utils/tcpros_utils.js

```diff
@@ -28,10 +28,11 @@ const md5Prefix = 'md5sum=';
 const topicPrefix = 'topic=';
 const servicePrefix = 'service=';
 const typePrefix = 'type=';
-const latchingPrefix = 'latching=';
-const persistentPrefix = 'persistent=';
 const errorPrefix = 'error=';
```

Comment:
Do you use any variable naming conventions in rosnodejs for (actual) constants? Like all CAPS or something similar?
I don't think this is an issue at all here, especially since it is consistent and these are all specifically named as Field's and Prefix's, just curious,

Reply:
I think I generally do CAPS_UNDERSCORE... this was one of the first things I wrote when I was still just messing around though so I probably wasn't thinking about it.